include/unifex/sender_concepts.hpp

```diff
@@ -67,10 +67,16 @@ template <typename Sender, typename Receiver>
 inline constexpr bool is_connectable_v =
   std::is_invocable_v<decltype(connect), Sender, Receiver>;
 
+template <typename Sender, typename Receiver>
+using is_connectable = std::is_invocable<decltype(connect), Sender, Receiver>;
+
```

Comment:
Ultimately, this will be called `sender_to`. Maybe call it that now.

Reply:
Should the metafunction-style names still be prefixed with `is_` to make space for the concept-definitions once we can support concepts?

I'm tempted to defer fixing the P0443-naming-compatibility aspects to a separate diff as this one is already getting quite large.